src/solvers/parabolic_solvers.jl

```diff
@@ -0,0 +1,35 @@
+"""
+  struct BR1 end
+
+The Bassi-Rebay flux from "A High-Order Accurate Discontinuous Finite Element Method for
+the Numerical Solution of the Compressible Navier-Stokes Equations" by Bassi, Rebay (1997).
+
+https://doi.org/10.1006/jcph.1996.5572
+"""
+struct BR1 end
```

Comment:
```suggestion
struct ViscousFluxBassiRebay1 end
const BR1 = ViscousFluxBassiRebay1 # optional
```
Just a suggestion to keep elixirs more readable for non-experts.